elasticsearch/src/main/java/org/apache/metamodel/elasticsearch/ElasticSearchDateConverter.java

```
@@ -0,0 +1,39 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.metamodel.elasticsearch;
+
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
+/**
+ * Util class to convert date strings from ElasticSearch to
+ * proper java Dates.
+ */
+final class ElasticSearchDateConverter {
+
+    public static Date tryToConvert(String dateAsString) {
+        try {
+            SimpleDateFormat dateFormat = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss.SSSX");
+            return dateFormat.parse(dateAsString);
+        } catch (ParseException e) {
+            return null;
```

I don't like to swallow an exception like this since it may give the wrong impression. Are we sure that this date format is always used by ES? If so I would throw an exception in this case. Or else at least make a loud warning or try the general purpose date converter or so.